servers/cromwell/README.md

```diff
@@ -15,15 +15,22 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
     export CROMWELL_URL=http://192.168.0.106:8000/api/workflows/v1
     ```
 
-2. Add a file named `config.json` to `job-manager/servers/cromwell/jobs` that contains the username and password for the specified cromwell instance (You may want to skip this step if the Cromwell instance does not have HTTP Basic Authentication):
+2. (Optional) By default, the shim layer uses **5** `sync` [Gunicorn workers](http://docs.gunicorn.org/en/stable/settings.html#worker-class), the number of workers is **5**. 
+    Job Manager also comes with `gevent` workers, you can override the default worker type and use dynamic number of workers(based on the number of CPU cores) by:
+    
+    ```
+    export GUNICORN_CMD_ARGS="--workers=$((2 * $(getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 2) + 1)) --worker-class gevent"
```

Comment:
Is `_NPROCESSORS_ONLN` available on all platforms? Mac OS X, Linux, etc.?

Reply:
Is our flask code compatible with gevent workers? Could you add some informatino explaining the functional difference between `sync` and `gevent`?

Also, I may separate these two concepts out. You can separately adjust the amount of workers and the worker-class so I might have these as two separate examples for clarity.

Reply:
@bfcrampton 

1. `_NPROCESSORS_ONLN` is available for most of the Linux systems (including MAC OS), `NPROCESSORS_ONLN` is the equivalence in BSD systems. If neither of them are found, the default value `2 x 2 + 1 = 5` will be used here. This command in the readme should be portable enough for users (who deploy JMUI) to use. What do you think?

2. According to the documentation (http://docs.gunicorn.org/en/stable/design.html#choosing-a-worker-type) and my understanding, I think our flask code is compatible with `gevent`(or `Eventlet`) workers. In fact, I think we should even avoid using `sync` workers because `sync` workers assume that:
    >  your application shouldn’t do anything that takes an undefined amount of time

    Among all of the async workers Gunicorn provides(`gevent/Eventlet`, `tornado`, `asyncIO`, `gthread`), I believe `gevent` should be the most compatible worker type for us to use.

3. Separating the 2 concepts out is a good idea, I will change it! Thanks!

Reply:
Interestingly, not sure if it's an illusion, I noticed a faster loading-speed when using `gevent` workers for JMUI. We should perform a benchmarking for this as well .

Reply:
Sorry I never responded to this, all sounds good. As a side note: if `gevent` seems faster maybe we should see if that can be made the default.